### PR TITLE
Added Helsinki-Systems Lexoffice plugin to the store

### PR DESCRIPTION
**Pull request by MariusGoehring**

### CODE REVIEW

File: _includes/store/helsinkisystems-lexoffice.md [hunk 1]
```diff
@@ -0,0 +1 @@
+{% include store/helsinkisystems-lexoffice.md %}
```
Comment (kevinpapst) [inline _includes/store/helsinkisystems-lexoffice.md:1]: This file must contain the entire product description.

Right now it references itself.

**Review by kevinpapst — CHANGES_REQUESTED** [review]

File: _data/de/store.yml [hunk 2]
```diff
@@ -146,6 +146,9 @@ items:
     helsinkisystems-budgetwatcher:
         title: Budget Watcher (beobachtet Budgets und versendet Benachrichtigungen)
         intro: Versendet Benachrichtigungen, wenn eine konfigurierbare Budgetgrenze bei einem Projekt erreicht ist
+    helsinkisystems-ledxoffice:
```
Comment (kevinpapst) [inline _data/de/store.yml:149]:
```suggestion
    helsinkisystems-lexoffice:
```

File: _data/store/helsinkisystems-lexoffice.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+demo: false
+gumroad: Lexoffice
+subscription: 99 €
+lemonsqueezy: "https://helsinkisystems.lemonsqueezy.com/checkout?cart=eef22eb9-992a-4d66-97a9-9abc4b04dc08"
+documentation: plugin-lexoffice
+bundle:
+    name: "Lexoffice"
+    purchase: true
+    versions:
+        - [ "2.0" ]
```
Comment (kevinpapst) [inline _data/store/helsinkisystems-lexoffice.yaml:12]: [ "plugin version", "min kimai version" ]

File: _includes/store/helsinkisystems-lexoffice.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+## Description
+
+Lexoffice plugin for Kimai is a plugin that allows you to automatically create invocies in Lexoffice, whenver an invoice in Kimai is created. By using the Lexoffice API (API key is needed) this plugin can create invoice drafts and finalized invoices in Lexoffice, based on your settings. Furthermore, this plugin allows you to customize the title and description of the items in the Lexoffice-invoice. The plugin therefore offers a few replacers you can choose from and that are replaced dynamically when creating the invoice for Lexoffice.
+
+## Installation
+
+This plugin does not need a database table, so there is no installation of the plugin necessary.
+To communicate with Lexoffice however, you need to install the Lexoffice-php-sdk composer package:
+``` composer require helsinki-systems/lexoffice-php-sdk:dev-master ```
+For lading the plugin, all you need to do after unzipping the plugin in the /var/plugins/ directory of your kimai installation, is to relaod the cache:
```
Comment (kevinpapst) [inline _includes/store/helsinkisystems-lexoffice.md:10]:
```suggestion
For loading the plugin, all you need to do after unzipping the plugin in the /var/plugins/ directory of your Kimai installation, is to reload the cache:
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
did you try these changes in jekyll?

File: _includes/store/helsinkisystems-lexoffice.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+## Description
+
+Lexoffice plugin for Kimai is a plugin that allows you to automatically create invocies in Lexoffice, whenver an invoice in Kimai is created. By using the Lexoffice API (API key is needed) this plugin can create invoice drafts and finalized invoices in Lexoffice, based on your settings. Furthermore, this plugin allows you to customize the title and description of the items in the Lexoffice-invoice. The plugin therefore offers a few replacers you can choose from and that are replaced dynamically when creating the invoice for Lexoffice.
+
+## Installation
+
+This plugin does not need a database table, so there is no installation of the plugin necessary.
+To communicate with Lexoffice however, you need to install the Lexoffice-php-sdk composer package:
+``` composer require helsinki-systems/lexoffice-php-sdk:dev-master ```
```
Comment (kevinpapst) [inline _includes/store/helsinkisystems-lexoffice.md:9]: Using composer will break the next update of Kimai.
People will pull and don't understand why the update doesn't work. 

You need to add a clear sentence about this expected problem and then link to https://www.kimai.org/documentation/updates.html#changed-files

File: _includes/store/helsinkisystems-lexoffice.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+For loading the plugin, all you need to do after unzipping the plugin in the /var/plugins/ directory of your Kimai installation, is to reload the cache:
```
Comment (kevinpapst) [inline _includes/store/helsinkisystems-lexoffice.md:10]: Installation instructions are auto-generated. 
Please remove that last sentence.

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Again: did you try this in jekyll locally?

File: _data/store/helsinkisystems-lexoffice.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+developer: helsinkisystems
+icon: fas fa-file-invoice
+demo: false
+gumroad: Lexoffice
```
Comment (kevinpapst) [inline _data/store/helsinkisystems-lexoffice.yaml:4]: Can people buy at Gumroad and Lemonsqueezy?

File: _data/store/helsinkisystems-lexoffice.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+subscription: 99 €
+lemonsqueezy: "https://helsinkisystems.lemonsqueezy.com/checkout?cart=eef22eb9-992a-4d66-97a9-9abc4b04dc08"
+documentation: plugin-lexoffice
```
Comment (kevinpapst) [inline _data/store/helsinkisystems-lexoffice.yaml:7]: this creates a link to a none existing documentation page

**Comment by MariusGoehring** [comment]
> Again: did you try this in jekyll locally?
yes, i tested it locally.

File: _includes/store/helsinkisystems-lexoffice.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+Lexoffice plugin for Kimai is a plugin that allows you to automatically create invocies in Lexoffice, whenver an invoice in Kimai is created. By using the Lexoffice API (API key is needed) this plugin can create invoice drafts and finalized invoices in Lexoffice, based on your settings. Furthermore, this plugin allows you to customize the title and description of the items in the Lexoffice-invoice. The plugin therefore offers a few replacers you can choose from and that are replaced dynamically when creating the invoice for Lexoffice.
+
+## Installation
+
+This plugin does not need a database table, so there is no installation of the plugin necessary.
+To communicate with Lexoffice however, you need to install the Lexoffice-php-sdk composer package:
+``` composer require helsinki-systems/lexoffice-php-sdk:dev-master ```
+
+Note: Using composer will break the next update of Kimai.
+People will pull and don't understand why the update doesn't work.
```
Comment (Scriptkiddi) [inline _includes/store/helsinkisystems-lexoffice.md:12]: this sentence does not belong here @MariusGoehring please fix

Comment (Scriptkiddi) [reply _includes/store/helsinkisystems-lexoffice.md:12]: On second though please change the paragraph to something like:
```
UPDATE WARNING: As this plugin uses composer it breaks on Kimai Updates.
You will have to rerun the composer installation step after each update. 
See https://www.kimai.org/documentation/updates.html#changed-files for more information
```

**Comment by MariusGoehring** [comment]
Rebased and tested on jekyll

File: _data/en/store.yml [hunk 10]
```diff
@@ -125,6 +125,9 @@ items:
     helsinkisystems-budgetwatcher:
         title: Budget Watcher (watch project budgets and send notifications)
         intro: Get notified whenever a certain amount of your projects budget is used
+    helsinkisystems-lexoffice:
+        title: Lexoffice (automatically create invoices in Lexoffice from invoices in Kimai)
```
Comment (kevinpapst) [inline _data/en/store.yml:129]: The title is something like "Lexoffice Invoice" or Lexoffice Invoice Sync" right?
The intro contains the description. 
Please remove the (rest) from the title and while you are on it: remove it from Budget watcher as well please in german and english.

File: _data/store/helsinkisystems-lexoffice.yaml [hunk 11]
```diff
@@ -0,0 +1,11 @@
+developer: helsinkisystems
+icon: fas fa-file-invoice
+demo: false
+subscription: 99 €
+lemonsqueezy: "https://helsinkisystems.lemonsqueezy.com/checkout?cart=eef22eb9-992a-4d66-97a9-9abc4b04dc08"
+bundle:
+    name: "Lexoffice"
```
Comment (kevinpapst) [inline _data/store/helsinkisystems-lexoffice.yaml:7]: The namespace Lexoffice is pretty generic. If it is only about Invoices, then please rename it to LexofficeInvoice

File: _includes/store/helsinkisystems-lexoffice.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+Lexoffice plugin for Kimai is a plugin that allows you to automatically create invocies in Lexoffice, whenver an invoice in Kimai is created. By using the Lexoffice API (API key is needed) this plugin can create invoice drafts and finalized invoices in Lexoffice, based on your settings. Furthermore, this plugin allows you to customize the title and description of the items in the Lexoffice-invoice. The plugin therefore offers a few replacers you can choose from and that are replaced dynamically when creating the invoice for Lexoffice.
+
+## Installation
+
+This plugin does not need a database table, so there is no installation of the plugin necessary.
+To communicate with Lexoffice however, you need to install the Lexoffice-php-sdk composer package:
+``` composer require helsinki-systems/lexoffice-php-sdk:dev-master ```
+
+UPDATE WARNING: As this plugin uses composer it breaks on Kimai Updates.
+You will have to rerun the composer installation step after each update.
```
Comment (kevinpapst) [inline _includes/store/helsinkisystems-lexoffice.md:12]: I think this is still not clear / I doubt that regular users will understand that. 
Maybe you can create `include/composer-update-warning.md` and add something like:

UPDATE WARNING: The next time you update Kimai you will see something like ... (show the merge message that will likely appear) what happens and you have to revert your changes first, then update Kimai and afterwards reinstall the plugin. Please read https://www.kimai.org/documentation/updates.html#changed-files

Then I can re-use this include on the LDAP docu page as well and we can update the shown warning on all related pages.

File: _store/helsinkisystems-lexoffice.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+---
+title: Lexoffice plugin (automatically create invoices in Lexoffice from Kimai invoices)
```
Comment (kevinpapst) [inline _store/helsinkisystems-lexoffice.md:2]:
```suggestion
title: Lexoffice Invoice Sync
```

or whatever you choose as title above

**Review by kevinpapst — CHANGES_REQUESTED** [review]

**Comment by MariusGoehring** [comment]
Okay have added the composer-update-warning.md. Also included it in _includes/store/helsinkisystems-lexoffice.md to be sure it is likned in there. If you have plans to automatically include it, tell me and i will remove the include from there

**Review by kevinpapst — APPROVED** [review]
Thanks 👍  I'll merge and make some smaller adjustments myself

**Pull request merged**